File: src/app/modelos/vehiculo.ts
export class Vehiculo {
    id_vehiculo:number;
    tipo:string;
    matricula:string;
    modelo:string;
    marca:string;

    constructor(id_vehiculo:number=0,tipo:string="",matricula:string="",modelo="",
    marca:string=""){
     this.id_vehiculo=id_vehiculo;
     this.tipo=tipo;
     this.matricula=matricula;
     this.modelo=modelo;
     this.marca=marca;
    }
}

File: src/main/java/model/vehiculo.java
package model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name="vehiculo")
public class vehiculo {

	@Id
	@GeneratedValue(strategy =GenerationType.IDENTITY)
	private int id_vehiculo ;
	@Column
	private String tipo ;
	@Column
	private String matricula ;
	@Column
	private String modelo ;
	@Column
	private String marca  ;
	
	public int getId_vehiculo() {
		return id_vehiculo;
	}
	public void setId_vehiculo(int id_vehiculo) {
		this.id_vehiculo = id_vehiculo;
	}
	public String getTipo() {
		return tipo;
	}
	public void setTipo(String tipo) {
		this.tipo = tipo;
	}
	public String getMatricula() {
		return matricula;
	}
	public void setMatricula(String matricula) {
		this.matricula = matricula;
	}
	public String getModelo() {
		return modelo;
	}
	public void setModelo(String modelo) {
		this.modelo = modelo;
	}
	public String getMarca() {
		return marca;
	}
	public void setMarca(String marca) {
		this.marca = marca;
	}
	
	
}
